java/src/jmri/jmrit/logix/BlockSpeedInfo.java

```diff
@@ -68,7 +69,7 @@ int getLastIndex() {
     @Override
     @Nonnull
     public String toString() {
-        StringBuilder sb = new StringBuilder("BlockSpeedInfo \"");
+        StringBuilder sb = new StringBuilder(" \"");
```

Comment:
This is a functional change. Is it intended?

Reply:
Yes, this method is called from SpeedUtil L947 , where the BlockSpeedInfo text is added to the debug log.